src/test/java/com/cloudant/tests/IndexTests.java

```diff
@@ -72,6 +70,16 @@ public void indexTestAll() {
         db.createIndex("Movie_year", "Movie_year", null,
                 new IndexField[]{new IndexField("Movie_year", SortOrder.asc)});
 
+        // selectorJson as a proper json object

```

Comment:
Your call on updating or removing (either one works for me) but since selectorJson no longer exists, the comment, in its current state, is confusing.